pyaedt/edb_core/hfss.py

```diff
@@ -942,9 +942,8 @@ def create_edge_port_horizontal(
     def create_lumped_port_on_net(
         self, nets=None, reference_layer=None, return_points_only=False, digit_resolution=6, at_bounding_box=True
     ):
-        """Create an edge port on nets. Only ports on traces (e.g. Path) are currently supported.
-        The command will look for traces on the nets and will try to assign vertical lumped port on first and last
-        point from the trace. To be used with cautious.
+        """Create an edge port on nets. The command will look for traces and polygons on the nets and will try to
+        assign vertical lumped port.
```

Comment:
```suggestion
        """Create an edge port on nets. This command looks for traces and polygons on the nets and tries to
        assign vertical lumped port.
```
The Google developer documentation style guide, which we fall, says to avoid future tense.